keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1743,55 +1702,41 @@ more-or-less **O(1)** behavior, so knowing the number of elements is
 not going to give you much information about how well the system is
 likely to be performing.
 
+(Update while going to Beta: it's still not clear.)
+
 - [X] Metrics
   - Metric names:
       - `network_programming_duration_seconds` (already exists)
       - `sync_proxy_rules_last_queued_timestamp_seconds` (already exists)
       - `sync_proxy_rules_last_timestamp_seconds` (already exists)
       - `sync_proxy_rules_duration_seconds` (already exists)
-      - ...
+      - `sync_proxy_rules_nftables_sync_failures_total`
+      - `sync_proxy_rules_nftables_cleanup_failures_total`
   - Components exposing the metric:
       - kube-proxy
 
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
-<!--
-Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
-implementation difficulties, etc.).
--->
-
-If we change any functionality relative to `iptables` mode (e.g., not
-allowing localhost NodePorts by default), it would be good to add
-metrics to the `iptables` mode, allowing users to be aware of whether
-they are depending on these features.
+We have now added some metrics to the `iptables` mode (e.g.,
+`kubeproxy_iptables_ct_state_invalid_dropped_packets_total`), allowing
+users to be aware of whether they are depending on features that work
+differently in the `nftables` backend, to help users decide whether
+they can migrate to `nftables`, and whether they need any non-standard
+configuration in order to do so.
```

Comment:
We should outline this somewhere, perhaps a feature blogpost?

Reply:
Yes, blog post sounds good